src/Epic/ContributionsEpicButtons.tsx

```diff
@@ -123,8 +124,10 @@ export const ContributionsEpicButtons = ({
     remindMeConfirmationText,
     remindMeConfirmationHeaderText,
     trackClick,
+    hidePaymentIcons,
 }: ContributionsEpicButtonsProps): JSX.Element => {
     const [sectionState, setSectionState] = useState<SectionState>('DEFAULT');
+    const showPaymentIcons = hidePaymentIcons != null && hidePaymentIcons === 'true' ? false : true;
```

Comment:
I wonder, can this be simplified to:

`const showPaymentIcons = !(hidePaymentIcons === 'true');`

? I.e. the only time they should be hidden is if `hidePaymentIcons` equals the string `'true'` and in all other cases show them, we we flip the result of compaing to `'true'`.

Reply:
Fixed with TomF suggested alternative